src/lib/findSpecificComponent.ts

```diff
@@ -0,0 +1,27 @@
+import { getNonShorthand } from "./getShorthand";
+
+const attributeMap: Record<string, Record<string, string>> = {
+  display: {
+    flex: "Flex",
+    grid: "Grid",
+  },
```

Comment:
@no-yan It works since using non-shorthand as key.

https://github.com/Monchi/eslint-plugin-chakra-ui/blob/require-specific-component-rule/src/lib/findSpecificComponent.ts#L18:L18

It would be broken in the future so I've added a test. Thank you for pointing it out!
[266844a](https://github.com/Monchi/eslint-plugin-chakra-ui/pull/29/commits/266844ac3c1a99c36b75678b3cb39bf20b19faaa)

Reply:
Thanks for adding the test. I was wrong. No problem!